package.json

```diff
@@ -98,6 +98,7 @@
     "react-native-web": "^0.13.12",
     "react-native-web-modal": "^1.0.1",
     "rimraf": "^3.0.0",
+    "svgs": "^4.1.0",
```

Comment:
Since we're removing `svgs`, do we need to add this?

Reply:
Of course not.